rootston/main.c

```diff
@@ -29,7 +30,11 @@ int main(int argc, char **argv) {
 	assert(server.wl_display = wl_display_create());
 	assert(server.wl_event_loop = wl_display_get_event_loop(server.wl_display));
 
-	assert(server.backend = wlr_backend_autocreate(server.wl_display));
+	//assert(server.backend = wlr_backend_autocreate(server.wl_display));
+	assert(server.backend = wlr_headless_backend_create(server.wl_display));
+	wlr_headless_add_output(server.backend, 1280, 720);
+	wlr_headless_add_input_device(server.backend, WLR_INPUT_DEVICE_KEYBOARD);
+	wlr_headless_add_input_device(server.backend, WLR_INPUT_DEVICE_POINTER);
```

Comment:
Shouldn't this part be commented out/reverted for now?
Not sure if it does any harm but it might confuse output lists/things like that.